MrTheGood/projectb_android
# Navigation bar

Changes:
 * Added working bottom navigation bar
 * Added colours
 * Added required fragments

How to test: 
 1. Run app
 2. Verify that clicking settings, map, search and memories works correctly
 3. Ignore that add memory doesn't work correctly

Screenshots:
![map](https://user-images.githubusercontent.com/23239376/54649852-265d7480-4aac-11e9-8c54-aac71790c761.png)
![memories](https://user-images.githubusercontent.com/23239376/54649854-26f60b00-4aac-11e9-9dba-9d9438360094.png)
![search](https://user-images.githubusercontent.com/23239376/54649855-26f60b00-4aac-11e9-9f4d-dfb7afd51dff.png)
![settings](https://user-images.githubusercontent.com/23239376/54649856-26f60b00-4aac-11e9-9a8c-e03e5724fdd0.png)
